source-dynamodb/.snapshots/TestDiscovery-additional_table_without_stream_enabled

```diff
@@ -0,0 +1,96 @@
+Binding 0:
+{
+    "recommended_name": "discoverTable",
+    "resource_config_json": {
+      "table": "discoverTable"
+    },
+    "document_schema_json": {
+      "$defs": {
+        "discoverTable": {
+          "type": "object",
+          "required": [
+            "partitionKey"
+          ],
+          "$anchor": "discoverTable",
+          "properties": {
+            "partitionKey": {
+              "type": "string"
+            }
+          }
+        }
+      },
+      "allOf": [
+        {
+          "required": [
+            "_meta"
+          ],
+          "properties": {
+            "_meta": {
+              "type": "object",
+              "required": [
+                "op"
+              ],
+              "properties": {
+                "approximateCreationDateTime": {
```

Comment:
`createdAt` ? Or `sourceCreatedAt` ? This name is a mouthful, its description can note that it's approximate, and `format` already conveys that it's a date-time.

Reply:
`createdAt` sounds good, the long-winded version was a direct copy from what DynamoDB's API calls it.

Reply:
Ah, in that case I rescind my comment, as it seems best not to re-define an existing concept. Happy to go either way though.